src/template.html

```diff
@@ -1,21 +1,87 @@
-<!DOCTYPE html>
-<html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-</head>
-
-<body>
-    <h1>title</h1>
-    <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Nobis expedita 
-        dignissimos laboriosam repellendus possimus cum, aut iusto vel hic 
-        3minima accusamus odio provident doloribus mollitia. 
-        Aliquam molestiae tempore tenetur ipsa, ullam totam eos 
-        nesciunt id minus nobis voluptates suscipit facilis? 
-        Reprehenderit nihil odit eos id!</p>
-</body>
-
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Montserrat:wght@800&display=swap"
+    rel="stylesheet" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+    integrity="sha512-1ycn6IcaQQ40/MKBW2W4Rhis/DbILU74C1vSrLJxCq57o941Ym01SwNsOMqvEBFlcgUa6xLiPY/NS5R+E6ztJQ=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <script src="https://kit.fontawesome.com/7cbe454c49.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <title>Javascript Capstone Project</title>
+</head>
+
+<body>
+
+  <!-- Navbar items -->
+  <nav>
+    <div id="navlist">
+      <a href="#">Delicious Meals</a>

```

Comment:
- [ ] You are almost there! Let's just add a counter in the navigation bar as the template shows, (this is a requirement, as you are a 3 members group):

|Template|Your design|
|---|---|
|![Captura de pantalla 2022-06-24 a la(s) 1 11 18 p  m](https://user-images.githubusercontent.com/83717285/175620092-0cbac315-2d1d-4627-8e2d-d88e92b30392.png)|![Captura de pantalla 2022-06-24 a la(s) 1 10 45 p  m](https://user-images.githubusercontent.com/83717285/175621616-18606b20-b996-4210-8136-8637f7acca15.png)|

Reply:
Done